packages/Sortable/src/Sortable.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uuidv4 from "uuid/v4";
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import SortableContext from "./SortableContext";
+import Item from "./components/Item/Item";
+import sortableStyles from "./Sortable.styles";
+
+const propTypes = {
+  children: PropTypes.node,
+  hasIndexes: PropTypes.bool,
+  onChange: PropTypes.func.isRequired,
+  onRemove: PropTypes.func,
+};
+
+const defaultProps = {
+  children: null,
+  hasIndexes: true,
+  onRemove: null,
+};
+
+function augmentChildren(children) {
```

Comment:
we have been using `extractChildren` or maybe this is more like cloneChidren?.

Reply:
Hmm... it's not really extracting in the same way you've used it.  And `clone` is a bit vague... it's really just "adding to" the children, which is why I think `augment` is still the best choice.